src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -656,7 +660,7 @@ export class AppComponent implements OnInit, AfterViewChecked {
     /**
      * Format all text to replace all new lines with spaces and performs HTML entity encoding
      */
-    textFormatter(row: number, cell: any, value: any, columnDef: any, dataContext: any): string {
+    static textFormatter(row: number, cell: any, value: DbCellValue, columnDef: any, dataContext: any): string {
```

Comment:
Had to make it static because slickgrid doesn't have reference to `this` or `self`